### PR TITLE
Added search icon that takes user to the Discover page 

### PR DESCRIPTION
**Pull request by codingwithlis**
## Jira Ticket 
[CFDP-1394]

## Description 
A "search" icon was added to the Nav bar that takes users to the Discover page

## Demo
Click on the Search icon on the Nav bar and it should take you to the Discover page 

## Notes 
Will come back later to make the search bar focused when landing on the Discover page 

## Images 
Screen Shot 2021-05-17 at 9 54 27 AM

[CFDP-1394]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1394

### CODE REVIEW

File: components/Nav/Nav.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ function Nav(props = {}) {
 
   return (
     <Styled>
+      <CustomLink href="/discover" color="fg">
```
Comment (asleepingpanda) [inline components/Nav/Nav.js:30]: @codingwithlis I think i would like for this to be to the right of the Menu Icon instead of the "Watch Now" button

**Review by asleepingpanda — CHANGES_REQUESTED** [review]

File: components/Nav/Nav.js (hunk 1 above)
Comment (codingwithlis) [reply components/Nav/Nav.js]: Let me know if this looks good!

Screen Shot 2021-05-17 at 3 09 40 PM

**Comment by dzwood** [comment]
@asleepingpanda @codingwithlis I'm thinking we should probably hide at least one of these buttons/icons on mobile. It looks a bit busy/messy to me. Let me know what you guys think though:

image

**Comment by asleepingpanda** [comment]
@dzwood I agree. I've gotten quite a bit of feedback from people on the navigation bar in general, so I think we push "pause" on that until we can get a more specifically defined issue for a "redesign"

**Review by asleepingpanda — APPROVED** [review]

**Pull request merged**